### PR TITLE
feat(css-prefix): inline support

### PR DESCRIPTION
**Pull request by ktfth**
## Summary

Included inline on the css-prefix

## Test Plan

rome test

### CODE REVIEW

**Comment by ematipico** [comment]
Thank you! Can you please rebase and run `./script generate-all-files`?

**Comment by ktfth** [comment]
> Thank you! Can you please rebase and run `./script generate-all-files`?

Done

**Comment by ktfth** [comment]
@ematipico All updated

File: internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/inline/input.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	inline: none;
+}
+
+.style {
+	border-inline-start: 1px;
+}
+
+.style {
+	border-inline-start: 2px dotted;
+}
```
Comment (ematipico) [inline internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/inline/input.css:11]: Is there a reason why you created two styles with the same property? One time should be enough

Comment (ktfth) [reply internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/inline/input.css]: To have some variations of the same property, i can clean that if you need

**Review by ktfth — COMMENTED** [review]
Changed

**Comment by jamiebuilds** [comment]
This looks great, thank you!

**Pull request merged**